src/tasks/tasks.py

```diff
@@ -1618,3 +1618,54 @@ def load_data(self, path, max_seq_len):
         self.val_data_text = list(val_data) + [masks[1]]
         self.test_data_text = list(te_data[:2]) + [te_targs] + [te_mask]
         log.info('\tFinished loading CCGTagging data.')
+
+@register_task('commitbank', rel_path='CommitmentBank/')
```

Comment:
Keep just one of these four or else add much clearer comments. Otherwise, this could throw off SuperGLUE users.